www/js/modules/main/components/histogram-container.js

```diff
@@ -1,40 +1,37 @@
-function histogramController($log) {
-    'ngInject';
+export default {
+    templateUrl: 'templates/components/main/_histogram-container.html',
```

Comment:
Inline templates suck. Use the provided template system. `www/templates/{module}/_{directive-name}.html`. Use underscore to denote partial html content.